WordPress/Classes/Networking/PlansRemote.swift

```diff
@@ -54,9 +56,173 @@ private func mapPlansResponse(response: AnyObject) throws -> (activePlan: Plan,
             return (result.0, plans)
         }
     })
+    
     guard let activePlan = parsedResponse.0 else {
         throw PlansRemote.Error.NoActivePlan
     }
     let availablePlans = parsedResponse.1.sort()
     return (activePlan, availablePlans)
 }
+
+private func parseFeatureGroups(json: [[String: AnyObject]], forPlan plan: Plan) {
+    let groups: [PlanFeatureGroup] = json.flatMap { groupJson in
+        guard let slugs = groupJson["items"] as? [String] else { return nil }
```

Comment:
I'd rather throw a `DecodeError` here. If a result from the API can't be parsed I'd rather learn about it as early as possible than ignore it.